### PR TITLE
Add SecOps docs to index

### PR DESCRIPTION
**Pull request by jleach**
## Summary
Add the Platform Services documentation for teams to use to create secure, Zero Trust, networks.

## Notable Changes

- Add to information and application security index.

### CODE REVIEW

File: app-web/registry/information-and-application-security.json [hunk 1]
```diff
@@ -46,6 +57,20 @@
...
         }
+      },
+      {
+        "sourceType": "github",
+        "sourceProperties": {
+          "url": "https://github.com/BCDevOps/platform-services/tree/master/security/aporeto/docs",
+          "owner": "BCDevOps",
+          "repo": "platform-services",
+          "files": [
+            "README.md",
```
Comment (patricksimonian) [inline app-web/registry/information-and-application-security.json]: are these files relative to `docs`?

the paths absolute from the root directory of the repo.

Comment (jleach) [reply app-web/registry/information-and-application-security.json]: ➜  docs git:(feature/devdocs) pwd
/Users/jleach/Workspace/bcdevx/platform-services/security/aporeto/docs
➜  docs git:(feature/devdocs) tree .
.
├── CustomPolicy.md
├── ExternalNetworks.md
├── QuickStart.md
├── README.md
├── add_labels.gif
├── add_nsp.gif
├── sample
│   └── quickstart-nsp.yaml
└── webview_nsp.gif

1 directory, 8 files
➜  docs git:(feature/devdocs)

Comment (jleach) [reply app-web/registry/information-and-application-security.json]: LMK if I need to change something.

Comment (patricksimonian) [reply app-web/registry/information-and-application-security.json]: yep should be based off the root directory

`security/aporeto/docs/CustomPolicy.md`

Comment (jleach) [reply app-web/registry/information-and-application-security.json]: Fixed.

**Comment by patricksimonian** [comment]
awesome. This is queued up for deployment.

**Pull request merged**